docker-proxy-junit-jupiter/src/integrationTest/resources/DockerProxyExtensionTest-services.yml

```diff
@@ -2,5 +2,6 @@ version: '2'
 
 services:
   webserver:
-    hostname: web.server.here
+    hostname: web
```

Comment:
No longer can resolve `web` when specifying the FQDN; must specify hostname/domainname separately